Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2003-11-02
Initial Package Version: 3.1.4
Origin: http://crux.fh-regensburg.de/cgi-bin/cvstrac/chngview?cn=2925
Description: Fixed u64 long long errors when building kdemultimedia
 
diff -Naur kdemultimedia-3.1.4.orig/kaudiocreator/cdconfigimp.cpp kdemultimedia-3.1.4/kaudiocreator/cdconfigimp.cpp
--- kdemultimedia-3.1.4.orig/kaudiocreator/cdconfigimp.cpp	2003-09-02 01:17:41.000000000 +0000
+++ kdemultimedia-3.1.4/kaudiocreator/cdconfigimp.cpp	2003-11-02 09:37:46.000000000 +0000
@@ -76,6 +76,7 @@
 /* This is in support for the Mega Hack, if cdparanoia ever is fixed, or we
    use another ripping library we can remove this.  */
 #define _LINUX_BYTEORDER_SWAB_H
+typedef unsigned long long __u64;
 #include <linux/cdrom.h>
 #include <sys/ioctl.h>
 
diff -Naur kdemultimedia-3.1.4.orig/kioslave/audiocd/audiocd.cpp kdemultimedia-3.1.4/kioslave/audiocd/audiocd.cpp
--- kdemultimedia-3.1.4.orig/kioslave/audiocd/audiocd.cpp	2003-09-02 01:17:44.000000000 +0000
+++ kdemultimedia-3.1.4/kioslave/audiocd/audiocd.cpp	2003-11-02 09:37:46.000000000 +0000
@@ -54,6 +54,7 @@
    fallbacks for non-GNUisms.  Fortunately we (or other headers) don't
    need the swab routines.  */
 #define _LINUX_BYTEORDER_SWAB_H
+typedef unsigned long long __u64;
 #include <linux/cdrom.h>
 #endif
 #include <sys/ioctl.h>
diff -Naur kdemultimedia-3.1.4.orig/kscd/libwm/plat_linux.c kdemultimedia-3.1.4/kscd/libwm/plat_linux.c
--- kdemultimedia-3.1.4.orig/kscd/libwm/plat_linux.c	2003-05-07 11:46:52.000000000 +0000
+++ kdemultimedia-3.1.4/kscd/libwm/plat_linux.c	2003-11-02 09:37:46.000000000 +0000
@@ -43,6 +43,8 @@
 #include <sys/wait.h>
 /* Try to get around bug #29274 */
 #include <linux/version.h>
+#undef __GNUC__
+typedef unsigned long long __u64;
 
 #include "include/wm_config.h"
 #include "include/wm_struct.h"
